lib/DidKeyDriver.js

```diff
@@ -119,10 +147,16 @@ export class DidKeyDriver {
     if(!did) {
       throw new TypeError('"did" must be a string.');
     }
-
     const [didAuthority, keyIdFragment] = did.split('#');
     const fingerprint = didAuthority.substring('did:key:'.length);
```

Comment:
We should stop calling this a `fingerprint` and just call it `publicKeyMultibase`. `fingerprint` is legacy.

Reply:
Updated.  Is this API still required and is `keyPair.fingerprint()` legacy as well? https://github.com/digitalbazaar/did-method-key/blob/main/lib/DidKeyDriver.js#L223-L226